### PR TITLE
DCOS-42589: DC/OS UI Package: design the package

### PR DESCRIPTION
**Pull request by GeorgiSTodorov**
Created a minimal package in the universe repository.

Jira: https://jira.mesosphere.com/browse/DCOS-42589

## Trade-offs
Icons and screenshots are placeholder strings, because the jira that adds them https://jira.mesosphere.com/browse/DCOS-41413 is not yet finished.

Build url is also placeholder, it will be changing as we bump the package version.

`marathon.json`, `config.json` and `command.json` are missing, because our package shouldn't be installed through universe (I used https://github.com/GeorgiSTodorov/universe/tree/version-3.x/repo/packages/D/dcos-core-cli/1000 as an example for this, it could be wrong).

This is my first time adding a package, tell me if I did something wrong.

### CODE REVIEW

**Comment by nLight** [comment]
@GeorgiSTodorov `[12:12:55][Step 2/2] dcos-registry: error: open dcos-ui-build-s3-url-placeholder: no such file or directory, try --help` looks like we need to provide at least something. Shall we put there the most recent version of dcos-ui?

File: repo/packages/D/dcos-ui/0/package.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "packagingVersion": "4.0",
+  "name": "dcos-ui",
+  "version": "1.0.0",
```
Comment (DanielMSchmidt) [inline repo/packages/D/dcos-ui/0/package.json]: Which version should we take here? Should it be the current one? (Not in this PR, but more of a general question?)

**Review by DanielMSchmidt — DISMISSED** [review]
Can't be installed, I would assume because we are missing a marathon.json. Or is there another reason?
	
bildschirmfoto 2018-10-01 um 12 11 00

**Comment by nLight** [comment]
@DanielMSchmidt it won't be installed via the Universe, only delivered. Scope of the issue was to provide a shell/template for our package. Though with tests in place I suppose we have to ship a meaningful package with all the URLs and a version in place.

File: repo/packages/D/dcos-ui/0/resource.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "images": {
+    "icon-small": "icon-small-placeholder.png",
```
Comment (nLight) [inline repo/packages/D/dcos-ui/0/resource.json]: @leemunroe do we have any timeline for the icons?

Comment (leemunroe) [reply repo/packages/D/dcos-ui/0/resource.json]: Default is fine since this is something users won't see

**Comment by GeorgiSTodorov** [comment]
@nLight I think Lee showed us some that we can some icons that are in the wiki, but I still don't have access to it. Can you please share links to the icons?

**Comment by nLight** [comment]
@GeorgiSTodorov let's remove images altogether for now. We've agreed to use default icons that we show when a package doesn't have any.

**Comment by GeorgiSTodorov** [comment]
@nLight removed them, let's see if the checks pass.

File: repo/packages/D/dcos-ui/0/package.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+  "packagingVersion": "4.0",
+  "name": "dcos-ui",
+  "version": "2.24.4",
+  "tags": ["dcos-ui", "dcos","ui"],
+  "maintainer": "frontend-dev@mesosphere.io",
+  "description": "The DC OS UI service.",
+  "scm": "https://github.com/dcos/dcos-ui",
+  "website": "https://github.com/dcos/dcos-ui/releases/tag/master+v2.24.4",
```
Comment (nLight) [inline repo/packages/D/dcos-ui/0/package.json]: Let's go with https://dcos.io/ for website

File: repo/packages/D/dcos-ui/0/package.json [hunk 4]
```diff
@@ -0,0 +1,11 @@
+{
+  "packagingVersion": "4.0",
+  "name": "dcos-ui",
+  "version": "2.24.4",
+  "tags": ["dcos-ui", "dcos","ui"],
+  "maintainer": "frontend-dev@mesosphere.io",
+  "description": "The DC OS UI service.",
```
Comment (nLight) [inline repo/packages/D/dcos-ui/0/package.json]: Let's change it to `DC/OS UI`

File: repo/packages/D/dcos-ui/0/package.json [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+  "scm": "https://github.com/dcos/dcos-ui",
+  "website": "https://github.com/dcos/dcos-ui/releases/tag/master+v2.24.4",
+  "framework": false
+}
```
Comment (nLight) [inline repo/packages/D/dcos-ui/0/package.json:12]: Let's add `"minDcosReleaseVersion" : "1.12"` field

**Comment by GeorgiSTodorov** [comment]
@nLight comments addressed.

**Review by nLight — APPROVED** [review]

**Comment by nLight** [comment]
🎉

**Review by DanielMSchmidt — APPROVED** [review]

**Comment by nLight** [comment]
@takirala hope we could just merge it when CI is green? 😅

**Pull request merged**